docs/workflows/collections.rst

```diff
@@ -256,22 +237,53 @@ config (e.g. ``~/.ansible.cfg`` or ``/etc/ansible/ansible.cfg``):
 .. code::
 
     [galaxy]
-    server: http://localhost/pulp_ansible/galaxy/my_content/
+    server_list = pulp
+
+    [galaxy_server.pulp]
+    server = http://localhost/pulp_ansible/galaxy/my_content/
+    username = my_user
+    password = my_password
 
-Then you can install without the ``-s`` url
+    [galaxy_server.deprecated]
+    server = http://localhost/pulp_ansible/galaxy/deprecated/
+    username = my_user
+    password = my_password
+
+Servers will be used in the order that they appear in ``server_list`` in the configuration file. Specific servers 
+can also be specified using the ``-s`` flag, either by their name or URL.
 
 .. code::
 
+   # Downloads from the "pulp" server
    ansible-galaxy collection install testing.ansible_testing_content:4.0.6
 
-   - downloading role 'elasticsearch', owned by elastic
-   - downloading role from http://localhost/pulp/content/dev/elastic/elasticsearch/6.2.4.tar.gz
-   - extracting elastic.elasticsearch to /home/vagrant/.ansible/roles/elastic.elasticsearch
-   - elastic.elasticsearch (6.2.4) was installed successfully
-
+   # Downloads from the "deprecated" server
+   ansible-galaxy collection install testing.ansible_testing_content:4.0.6 -s deprecated
 
 .. _collection-publish:
 
+Install a Collection, hosted by Pulp, using ``ansible-galaxy``
+--------------------------------------------------------------
+
+You can use `ansible-galaxy` to install a collection by its namespace and name from pulp_ansible
+using the `install` command. For example to install the `hello` collection from above into a
+directory `~/collections` you can specify:
+
+``ansible-galaxy collection install testing.ansible_testing_content -c``
+
+
+This assumes that the `hello` Collection is being served by the Distribution `ansible-galaxy` is
+configured to use.
+
+Using a specific version
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Install your collection by name by specifying the distribution serving your Repository's content using the
+``-s`` option.
+
+``ansible-galaxy collection install testing.ansible_testing_content:4.0.6 -c``
+
+
```

Comment:
This whole section confused me. Also i think it is misplaced below the `collection-publish` anchor.

Reply:
I should elaborate a bit. To me the command examples are not reflecting what the text around promises. Also what is the `-c`?